make/photon/db/rpm_builder.sh

```diff
@@ -0,0 +1,27 @@
+set -e
+
+name='postgresql'
+version='9.6.21'
+
+cur=$PWD
+
+workDir='pg_builder'
+rm -rf $workDir && mkdir -p $workDir && cd ./$workDir
+
+# step 1: get source code of pg 9.6, and rename the code directory from postgres to postgres96
+wget http://ftp.postgresql.org/pub/source/v$version/$name-$version.tar.bz2
+bzip2 -d ./$name-$version.tar.bz2 && tar -xvf ./$name-$version.tar
+mkdir -p ${name}96-$version && cd ./$name-$version && cp -rf ./* ../${name}96-$version && cd .. && rm -rf ./$name-$version
+tar -cvjSf ${name}96-$version.tar.bz2 ${name}96-$version
```

Comment:
1)  just `cp -r ./$name-$version/* ./${name}96-$version/` and  you don't need `cd ./$name-$version` and `cd ..`
2) why sometimes use `./` sometime does not?